lib/selectivity/rspec.rb

```diff
@@ -37,11 +37,12 @@ def selectivity_unselect(value, *args)
       private
 
       def find_selectivity_input(from, options)
-        find(:div, from, options)
-      rescue Capybara::ElementNotFound
-        label = find('label', { text: from }.merge(options))
-
-        find(:div, "##{label[:for]}", options)
+        label = first(:xpath, ".//label[contains(., '#{from}')]", options)
+        unless lable.nil?
```

Comment:
1. It's much more readable to use `if ... else` than `unless ... else`
2. You have a typo here.